docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb

```diff
@@ -30,7 +24,7 @@
   {
```

Comment:
Hi,@xinwenh , please pay attention to the specification of punctuation, sometimes it is `、 `, sometimes `, `, it seems particularly unprofessional